src/Native/Runtime/gcheaputilities.h

```diff
@@ -20,6 +20,12 @@ GPTR_DECL(uint32_t,g_card_table);
 }
 #endif // !DACCESS_COMPILE
 
+// For single-proc machines, the EE will use a single, shared alloc context
```

Comment:
CoreRT does not use the global GC alloc context. `g_global_alloc_context` should not be needed in CoreRT. Is it possible to delete it?